### PR TITLE
chore: Add QE CLI flag

### PR DESCRIPTION
**Pull request by jamesclark-Zapata**
# The problem

We want to swap the defaults for logging in soon, however we don't have a QE flag.

# This PR's solution

* Add `--qe` option for logging in, mutually exclusive with `--ce`.
* Restructure CLI code to pass a `RuntimeName` instead of "is CE" bool.

# Checklist

_Check that this PR satisfies the following items:_

- [x] Tests have been added for new features/changed behavior (if no new features have been added, check the box).
- [x] The [changelog file](CHANGELOG.md) has been updated with a user-readable description of the changes (if the change isn't visible to the user in any way, check the box).
- [x] The PR's title is prefixed with `<feat/fix/chore/internal/docs>[!]:`
- [x] The PR is linked to a JIRA ticket (if there's no suitable ticket, check the box).

### CODE REVIEW

File: src/orquestra/sdk/_base/cli/_dorq/_entry.py [hunk 1]
```diff
@@ -405,20 +407,28 @@ def status():
...
     """
     Login in to remote cluster
     """
     from ._login._login import Action
 
+    runtime_name: RemoteRuntime
+    if ce:
+        runtime_name = RuntimeName.CE_REMOTE
+    else:
+        runtime_name = RuntimeName.QE_REMOTE
```
Comment (alexjuda) [inline src/orquestra/sdk/_base/cli/_dorq/_entry.py:428]: **nitpick**: so far we wanted to keep the `_entry` module as slim as possible, in particular devoid of any logic that would need to be tested. This ce/qe seems like a good candidate for an argument resolver. WDYT about moving it there?

Comment (jamesclark-Zapata) [reply src/orquestra/sdk/_base/cli/_dorq/_entry.py]: Seems like a lot of boilerplate, I moved it to `click`'s code via the `flag_value` kwarg in aa7d501. Does that seem better?

What we lose is the `cloup` mutually exclusive support. You can only have a single value for `runtime_name` but `orq login -s ... --ce --qe` would use `--qe` as it was the last option specified.

Comment (jamesclark-Zapata) [reply src/orquestra/sdk/_base/cli/_dorq/_entry.py]: bump

Comment (alexjuda) [reply src/orquestra/sdk/_base/cli/_dorq/_entry.py]: > What we lose is the cloup mutually exclusive support.

Yeah that's a good point. Yikes. Pre- 3facd7afdbfaf9d8878c5489443f2b4b43dbc240 seems like a better solution after all.

**Review by alexjuda — APPROVED** [review]

**Pull request merged**